src/deviceconfig.cpp

```diff
@@ -60,18 +60,7 @@ DeviceConfig::DeviceConfig()
     {
         debugW("DeviceConfig could not be loaded from JSON, using defaults");
 
-        // Set default for additional settings in this code
-        location = cszLocation;
-        locationIsZip = bLocationIsZip;
-        countryCode = cszCountryCode;
-        openWeatherApiKey = cszOpenWeatherAPIKey;
-        use24HourClock = false;
-        useCelsius = false;
-        ntpServer = NTP_SERVER_DEFAULT;
-        rememberCurrentEffect = true;
-        powerLimit = POWER_LIMIT_DEFAULT;
-

```

Comment:
All this was largely a duplication of defaults set in the DeviceConfig declaration in deviceconfig.h. Whatever was missing there has now been added there, so this can go.